src/Command/AddMassMediaCommand.php

```diff
@@ -104,8 +122,8 @@ protected function insertMedia(array $data, OutputInterface $output)
 
     protected function optimize()
     {
-        if ($this->getContainer()->has('doctrine')) {
-            $this->getContainer()->get('doctrine')->getManager()->getUnitOfWork()->clear();
+        if (null !== $this->doctrine) {
```

Comment:
can this be null?

Reply:
I guess mongodb can be used, but in that case the definition should be modified.